tools/data_integrity/score_instrument.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * this tools scores any registered instrument that was build using the
+ * NDB_BVL_Instrument class and that has a working score() method.
+ * the cmd-line arguments need to contain a valid test_name and 'all' or 'one' and a
+ * candid and SessionID 'all' scores all records of an instrument that belong to an
+ * active timepoint and whose data entry is completed and administration <> None.
+ * the non-administered records' scores have to be reset in the DB.
```

Comment:
I don't understand that last sentence? Also, start sentence with upper case letters and there should not be acronyms in the documentation (DB = database).